### PR TITLE
BUG: Fix version check error for Pandas release candidates.

### PR DESCRIPTION
**Pull request by nkeim**
Discovered while investigating #282. Apprently `StrictVersion` is… strict.

### CODE REVIEW

**Comment by caspervdw** [comment]
:+1: It might make sense to move the bit of code to the top, so make a global `is_recent_pandas`.

**Comment by nkeim** [comment]
Good idea. The test also contained a version check; I've now fixed the code there also. And since the test was in two places, I moved it to `utils.py`.

File: trackpy/utils.py [hunk 1]
```diff
@@ -16,6 +17,14 @@
 
 import trackpy
 
+# Set is_pandas_recent for use elsewhere.
+# Pandas >= 0.16.0 lets us check if a DataFrame is a view.
+try:
+    is_pandas_recent = (StrictVersion(pd.__version__) >=
+                        StrictVersion('0.16.0'))
+except ValueError:  # Probably a release candidate
```
Comment (tacaswell) [inline trackpy/utils.py]: where does the valueerror come from?

Comment (nkeim) [inline trackpy/utils.py]: From the "rc" in the version string. We could use `LooseVersion`, which allows alphabetic characters. But I figured that the test needed a fallback anyhow, so I went for that.

**Comment by danielballan** [comment]
Good enough!

**Pull request merged**